bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java

```diff
@@ -262,7 +263,7 @@ protected void doGet(@NotNull SlingHttpServletRequest request, @NotNull SlingHtt
                 metrics.markImageStreamed();
             }
         } catch (IllegalArgumentException e) {
-            LOGGER.error("Invalid image request", e.getMessage());
+            LOGGER.error("Invalid image request" + e);
```

Comment:
I'd change to: `LOGGER.error("Invalid image request", e);`

Reply:
Thanks for comment, should we change to `LOGGER.error("Invalid image request {}", e);` So the `e` can be log to log file

Reply:
@leonso , @jckautzmann has the same suggestion I had in https://github.com/adobe/aem-core-wcm-components/pull/2096#discussion_r829984889. https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/Logger.html#error-java.lang.CharSequence-java.lang.Throwable- will also log the stacktrace when passing the throwable as a second parameter.